src/utils/resolve-project-base-path.ts

```diff
@@ -5,9 +5,10 @@ import { ProjectType } from '../config';
 
 const angularConfig = ['angular.json', '.angular.json'];
 const workspaceConfig = ['workspace.json'];
+const projectConfig = ['project.json'];
 
-function searchConfig(searchPlaces: string[]) {
-  return cosmiconfigSync('', { searchPlaces }).search()?.config;
+function searchConfig(searchPlaces: string[], searchFrom: string = '') {
```

Comment:
When providing a default value the type is inffered.
```suggestion
function searchConfig(searchPlaces: string[], searchFrom = '') {
```